src/main/java/uk/ac/ebi/ena/readtools/common/reads/normalizers/htsjdk/IlluminaQualityNormalizer.java

```diff
@@ -0,0 +1,12 @@
+package uk.ac.ebi.ena.readtools.common.reads.normalizers.htsjdk;
+
+import htsjdk.samtools.util.SolexaQualityConverter;
+import uk.ac.ebi.ena.readtools.common.reads.QualityNormalizer;
+
+public class IlluminaQualityNormalizer implements QualityNormalizer {
+
+    @Override
+    public void normalize(byte[] qualities) {
+        SolexaQualityConverter.getSingleton().convertSolexa_1_3_QualityCharsToPhredBinary(qualities);

```

Comment:
Looks ok. Just double checking. Notes to myself:

convertSolexa_1_3_QualityCharsToPhredBinary(byte[] solexaQuals)
-----------------------------
Casava 1.3 stores phred-scaled qualities, but non-standard because they have 64 added to them rather than the standard 33.

corresponds to I think:

https://en.wikipedia.org/wiki/FASTQ_format#Quality I would expect this class to be called SolexaQualityNormalizer
-----------------------------
Starting with Illumina 1.3 and before Illumina 1.8, the format encoded a Phred quality score from 0 to 62 using ASCII 64 to 126 (although in raw read data Phred scores from 0 to 40 only are expected).
and
Starting in Illumina 1.5 and before Illumina 1.8, the Phred scores 0 to 2 have a slightly different meaning...